hack/stats.sh

```diff
@@ -50,5 +50,7 @@ mkdir -p stats
         "downloads": .downloadCount
         }
     ]
-    }'
+    }' || {
+        :
+    }
```

Comment:
Actually this could be a bit simpler
```suggestion
    }' || true
```

Reply:
Good call, added!